Library/Formula/virtuoso.rb

```diff
@@ -1,12 +1,11 @@
-require 'formula'
-
 class Virtuoso < Formula
-  homepage 'http://virtuoso.openlinksw.com/wiki/main/'
-  url "https://downloads.sourceforge.net/project/virtuoso/virtuoso/7.1.0/virtuoso-opensource-7.1.0.tar.gz"
-  sha1 "255d275d810bdb7cfa55fef46517724823c4c561"
+  homepage "http://virtuoso.openlinksw.com/wiki/main/"
+  url "https://github.com/openlink/virtuoso-opensource/releases/download/v7.2.0.1/virtuoso-opensource-7.2.0_p1.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/virtuoso/virtuoso/7.2.0/virtuoso-opensource-7.2.0_p1.tar.gz"
+  sha256 "fe0fc64c169fc59322142d4e4a90d1328f7063a07f9d9035aaf5adc9dde28302"
```

Comment:
Could you please add a `version "7.2.0-1"`? Homebrew fails to get the correct version here.